### PR TITLE
Nate2

### PR DESCRIPTION
**Pull request by nholdsworth**
Hey so I added some user friendly-ness here, and made the wine paring search more efficient and began working on an error handling for when the cooking instructions comes back null.  Please approve me

### CODE REVIEW

**Review by xoamara — DISMISSED** [review]

File: assets/javascript/script.js [hunk 1]
```diff
@@ -178,6 +200,17 @@ $(document).ready(function () {
             let results = response;
             console.log(results);
 
+<<<<<<< HEAD
```
Comment (Heidijvr) [inline assets/javascript/script.js]: It looks like you committed a file with a merge conflict

Comment (xoamara) [reply assets/javascript/script.js]: Odd thing is github is not saying it is a merge conflict...there would be a resolution button if so

Comment (xoamara) [reply assets/javascript/script.js]: Jason said this means that a merge marker got committed to the source code and that we should fix it.  No idea how though.  Going to have to look it up.

Comment (nholdsworth) [inline assets/javascript/script.js]: conflicts are not a big deal you guys I have had a conflict every time I have pulled practically . I had to resolve conflicts before I could pull Neri's code.  But anyway, I will pull the other two merges and then push this again...

**Pull request closed without merging**